Ui/Component/Listing/Column/MenuList/PageActions.php

```diff
@@ -10,17 +10,23 @@ public function prepareDataSource(array $dataSource)
         if (isset($dataSource["data"]["items"])) {
             foreach ($dataSource["data"]["items"] as & $item) {
                 $name = $this->getData("name");
-                $id = "X";
-                if (isset($item["menu_id"])) {
-                    $id = $item["menu_id"];
-                }
+                $id = $item["menu_id"] ?? "X";
```

Comment:
I also like using the `??` but I have seen that in main composer.json, minimum requirement for the menu module is: `"magento/framework": "100.1.*|101.0.*|102.0.*|103.0.*"`, which is Magento 2.1.x, still compatible with php 5.6, https://github.com/magento/magento2/blob/2.1/composer.json#L11

So I think we should add a constraint in snowdog menu's composer.json to restrict PHP version to be equal or greater than 7.x